nbbuild/antsrc/org/netbeans/nbbuild/AutoUpdateCatalogParser.java

```diff
@@ -240,6 +241,8 @@ public void endElement (String uri, String localName, String qName) throws SAXEx
                 break;
             case manifest :
                 break;
+            case message_digest:
```

Comment:
The `AutoUpdate` task was very verbose with modern catalogs. Assuming this field got in with one of @matthiasblaesing  fixes, I've added it to the set of recognized, but not processed fields.